package.json

```diff
@@ -37,7 +37,7 @@
         "process:sass": "node-sass-chokidar ./kuma/static/styles/ -o ./static/styles -w",
         "start": "run-p process:sass webpack:dev",
         "test": "jest --testPathIgnorePatterns tests/sass",
-        "test-ci": "JEST_JUNIT_OUTPUT_DIR='./test_results' JEST_JUNIT_OUTPUT_NAME='jest.xml' jest --ci --reporters=default --reporters=jest-junit --testPathIgnorePatterns tests/sass",
+        "test-ci": "JEST_JUNIT_OUTPUT_DIR='./test_results' JEST_JUNIT_OUTPUT_NAME='jest.xml' NODE_ICU_DATA=node_modules/full-icu jest --ci --reporters=default --reporters=jest-junit --testPathIgnorePatterns tests/sass",
```

Comment:
The tests are passing locally, but in Docker these two are failing, I think because there is no `Intl` support in the Docker container:
```
FAIL kuma/javascript/src/payments/formatDate.test.js
  ● formatDate() › formats date according to locale
    expect(received).toEqual(expected) // deep equality
    Expected: "22 de maio de 2020"
    Received: "May 22, 2020"
      10 |         const mockDate = 1590149080000;
      11 |         const expected = '22 de maio de 2020';
    > 12 |         expect(formatDate('pt-PT', mockDate)).toEqual(expected);
         |                                               ^
      13 |     });
      14 |     it('formats date according to options', () => {
      15 |         const mockDate = '2020-05-23T08:04:40';
      at Object.<anonymous> (kuma/javascript/src/payments/formatDate.test.js:12:47)
  ● formatDate() › formats date according to options
    expect(received).toEqual(expected) // deep equality
    Expected: "05.20"
    Received: "05/20"
      19 |         };
      20 |         const expected = '05.20';
    > 21 |         expect(formatDate('tr', mockDate, mockOptions)).toEqual(expected);
         |                                                         ^
      22 |     });
      23 |     it('returns empty string for invalid date', () => {
      24 |         const mockDate = '11/9999';
      at Object.<anonymous> (kuma/javascript/src/payments/formatDate.test.js:21:57)
```
I added a devDependency `full-icu` to fix it, but it seems like there could be a better solution. I don't like having to add a package just to get tests to pass on Docker-- is there a way to add `Intl` support to the container somehow?

Reply:
This seems worthwhile to fix. So you can't use `new Intl()` inside the docker container??
What does Google have to say on this?